src/main/java/Car.java

```diff
@@ -1,15 +1,15 @@
 // Car class
 public class Car {
     // Instance variables
-    private String make;
+    private String brand;
     private String model;
     private String color;
     private int year;
     private boolean running;
 
     // Constructor
-    public Car(String make, String model, String color, int year) {
-        this.make = make;
+    public Car(String brand, String model, String color, int year) {
```

Comment:
Verify is this doesn't cause a null pointer.

Reply:
I verify and no null pointer happening